persistence/relational-jdbc/src/main/java/org/apache/polaris/persistence/relational/jdbc/JdbcMetaStoreManagerFactory.java

```diff
@@ -143,15 +146,16 @@ public synchronized Map<String, PrincipalSecretsResult> bootstrapRealms(
       RealmContext realmContext = () -> realm;
       if (!metaStoreManagerMap.containsKey(realm)) {
         DatasourceOperations datasourceOperations = getDatasourceOperations();
-        try {
-          // Run the set-up script to create the tables.
-          datasourceOperations.executeScript(
-              datasourceOperations
-                  .getDatabaseType()
-                  .openInitScriptResource(bootstrapOptions.schemaOptions()));
-        } catch (SQLException e) {
-          throw new RuntimeException(
-              String.format("Error executing sql script: %s", e.getMessage()), e);
+        if (bootstrapOptions.schemaOptions().setupSchema()) {
+          setupSchema(bootstrapOptions.schemaOptions());
```

Comment:
We are doing this setup per realm, but the schema cannot change from one realm to another. I think this should be done in a startup method. This would be beneficial to both the admin tool and the server imho.

Reply:
If we want to support a schemas-per-realm kind of deployment (which is not the case today), then we'd need another config property to indicate which kind of schema we are using, and when to set it up (at startup vs for each realm).